index.js

```diff
@@ -1,15 +1,13 @@
 // Set up auth for Google spreadsheet
-var GoogleSpreadsheet = require('google-spreadsheet');
-var creds = require('./google-sheets-token.json');
-const { promisify } = require('util');
+const GoogleSpreadsheet = require('google-spreadsheet')
+const creds = require('./google-sheets-token.json')
+const { promisify } = require('util')
 
 // Create a document object using the ID of the spreadsheet - obtained from its URL.
-const SPREADSHEET_ID = '1feRnLYA4KYoe6NLtg50w3e33rYZpCJtS5cYhIo-YH38';
+const SPREADSHEET_ID = '1feRnLYA4KYoe6NLtg50w3e33rYZpCJtS5cYhIo-YH38'
 const SPREADSHEET_TITLE = 'Week-by-Week: Initiative Status'
-var data = 0
 // Todo: grab by the tracking URL instead
-var feature = 'MVP feature 2'
-var status_colors = {
+const STATUS_COLORS = {
```

Comment:
Tiny consistency change: global constants should be uppercase. Just personal preference.

Reply:
Ah, like a Ruby constant 👍